src/app/cube/details/details.component.ts

```diff
@@ -168,6 +203,76 @@ export class DetailsComponent implements OnInit, OnDestroy {
       }
       console.log(e);
     }
+
+    if (this.hasRevisions) {
```

Comment:
Between getting the released object and the revised one, is there a way to put this logic into a separate method so that this logic isn't repeated too much?

Reply:
Moved this to a separate function.